msticpy/context/tiproviders/mblookup.py

```diff
@@ -0,0 +1,317 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# Author: Thomas Roccia - @fr0gger_
+# --------------------------------------------------------------------------
+"""
+MalwareBazaar Provider.
+
+Input can be a hash, tag, signature, filetype, clamav signature, imphash, dhash,
+tlsh, telfhash, certificate issuer info, certificate subject info, or certificate.
+
+The class can be used to:
+    - Get data about a sample from MalwareBazaar
+    - Download malware sample from MalwareBazaar
+    - Query Code Signing Certificate Blocklist (CSCB)
+    - Get the recent addition from MalwareBazaar
+
+"""
+from enum import Enum
+from typing import Set
+
+import json
+import requests

```

Comment:
We've tried to standardize on using httpx for web requests - are you able to use that package here instead?

Reply:
Perfect I'll do that asap